api/bases/test.openstack.org_tempests.yaml

```diff
@@ -1599,6 +1599,13 @@ spec:
                       executed with --verbose
                     type: boolean
                 type: object
+              timingData:
+                default: false
+                description: |-
+                  A boolean that enables saving stestr timing data to a persistent volume.
+                  When true, subsequent Tempest test runs save time by reusing this data
+                  to optimize the test order.
+                type: boolean
```

Comment:
Thinking now about it, I think we will be passing here an URL where to find the timing data, instead.

After my recent patch https://github.com/openstack-k8s-operators/tcib/pull/320 the timing data themselves will be collected in logs ~ so we can simply have post-run playbook in CI jobs that will update the timing data in successful runs. In that case, the logic in run-tempest.sh will be much simpler.